lib/determine-basal/determine-basal.js

```diff
@@ -125,19 +125,22 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
     } 
     if (eventualBG < profile.min_bg) { // if eventual BG is below target:
         rT.reason = "Eventual BG " + eventualBG + "<" + profile.min_bg;
-        // if 5m or 15m avg BG is rising faster than expected delta
         if (minDelta > expectedDelta) {
-            if (glucose_status.delta > glucose_status.avgdelta) {
-                rT.reason += ", but Delta " + tick + " > Exp. Delta " + expectedDelta;
-            } else {
-                rT.reason += ", but Avg. Delta " + glucose_status.avgdelta.toFixed(2) + " > Exp. Delta " + expectedDelta;
-            }
-            if (currenttemp.duration > 0) { // if there is currently any temp basal running
-                rT.reason = rT.reason += "; cancel";
-                return setTempBasal(0, 0, profile, rT, offline); // cancel temp
-            } else {
-            rT.reason = rT.reason += "; no temp to cancel";
-            return rT;
+            if (minDelta > 0) { // if 5m or 15m avg BG is rising faster than expected delta
```

Comment:
I was trying to split up "rising faster" vs. "falling slower". Suggestions/edits welcome.